src/utilities/formatTimestamp.ts

```diff
@@ -1,4 +1,6 @@
 const formatTimestamp = (entry: string, isEdited?: boolean) => {
+  if (!entry) return "N/A";
```

Comment:
>I'm thinking if in the future more cases pop up where we have to handle empty timestamps, then maybe we can deal with that when it comes. Do you agree?

Yeah I'm cool with this. This solution felt kinda weird to implement tbh because we didn't do it with alot of the other functions, but I agree if there are too many implementations of the conditional render then we should look to change the solution